lib/internal/Magento/Framework/Module/Registrar.php

```diff
@@ -32,6 +32,11 @@ public static function registerModule($moduleName, $path)
         self::$modulePaths[$moduleName] = $path;
     }
 
+    public static function unregisterModule($moduleName)
```

Comment:
what about adding another Modul Registrar Class which does not make use of static. The static method/the adding of modules is not part of the interface. So its also easier to Test with it.